src/Pages/sanghunlee/Login/LoginContent.js

```diff
@@ -0,0 +1,102 @@
+import React from'react';
+import {withRouter, Link} from 'react-router-dom';
+
+
+class LoginContent extends React.Component{
+    constructor(props){
+        super(props)
+
+        this.state ={
+            id: '',
+            pw: '',
+            button: 'button',
+            disabled: true,
+        }
+
+    }
+
+    updateId =(evt)=>{
+        this.setState({id : evt.target.value})
+        if(this.state.id.length >=5 && this.state.pw.length>=5 && this.state.id.includes("@")){
+            this.setState({disabled : false});
+        }
+        else{
+            this.setState({disabled : true});
+        }
+    }
+    
+    updatePw =(e)=>{
+        this.setState({pw:e.target.value})
+        if(this.state.id.length >=5 && this.state.pw.length>=5 && this.state.id.includes("@")){
+            this.setState({disabled : false});
+        }
+        else{
+            this.setState({disabled: true});
+        }
+    }
+
+
+    // LoginButton = (evt)=>{
+    //     console.log(this.state.id)
+    //     console.log(this.state.pw)
+    // }
+
+
+    render(){
+        return(<article>
+            <div className="login">
+                <div className="loginContentWrapper">
+                    <img alt ='' className="instaLogo" src="../images/sanghunlee/logo_text.png" />
+                    <input
+                    value ={this.state.id} 
+                    onChange ={this.updateId} 
+                    className="inputId" 
+                    type="text" 
+                    placeholder="Phone number, username, or email"/>
```

Comment:
```suggestion
                    <input
                        value ={this.state.id} 
                        onChange ={this.updateId} 
                        className="inputId" 
                        type="text" 
                        placeholder="Phone number, username, or email"
                    />
```

- 들여쓰기 전체적으로 검토해서 수정 부탁드립니다🙇🏻‍♂️ 코드는 읽히는 경우가 쓰는 경우보다 많기 때문에 가독성 너무 중요합니다.

Reply:
한 요소 안에 여러 속성을 부여하는 경우에도 우선순위 고려해서 순서 정해주세요. 보통 className을 먼저 씁니다. 참고하세요!